static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js

```diff
@@ -8,6 +8,7 @@ define([
     'raven',
     'common/utils/$',
     'common/utils/config',
+    'common/utils/fastdom-promise',
```

Comment:
If we are not using the promise, do we still need the `fastdom-promise` dependency?